fix(docs): made the docs consistent for all frameworks similar to astro (#371)
getsentry/spotlight: packages/website/src/content/docs/setup/other.mdx

---

> ## 3. Run the Sidecar

Reviewer: Just noticed (can't mark the line though in the GH UI): Can we remove the `:::tip` block in lines 91-103? I think this is essentially a duplicate of the newer "Leverage Sentry SDKs" section

Author: Done, removed. the Leverage Sentry SDK section is fine. no need of this tip.

</CardGrid>

## Bonus: Leverage Sentry SDKs

> To get the most value out of Spotlight, configure Sentry's SDKs. You don't need to be using the Sentry service to take advantage of Sentry's SDKs to power Spotlight.

Reviewer: ```suggestion
To get the most value out of Spotlight, configure Sentry's SDKs. You don't have to use sentry.io but you can use the them to just power Spotlight.
```
